### PR TITLE
SAK-42895 BaseResourceProperties: Clean up instances

### PR DESCRIPTION
**Pull request by axxter99**

### CODE REVIEW

File: kernel/kernel-util/src/main/java/org/sakaiproject/util/BaseResourceProperties.java [hunk 1]
```diff
@@ -137,7 +137,7 @@ public BaseResourceProperties(Element el)
 					// if we don't have a value yet, make a list to hold this one
 					if (current == null)
 					{
-						List values = new Vector();
+						List<String> values = new Vector();
```
Comment (adrianfish) [inline kernel/kernel-util/src/main/java/org/sakaiproject/util/BaseResourceProperties.java]: This should be new Vector<>() to turn on inference

File: kernel/kernel-util/src/main/java/org/sakaiproject/util/BaseResourceProperties.java [hunk 2]
```diff
@@ -458,24 +468,14 @@ public long getLongProperty(String name) throws EntityPropertyNotDefinedExceptio
...
-	 */
+	@Override
 	public Time getTimeProperty(String name) throws EntityPropertyNotDefinedException, EntityPropertyTypeException
 	{
 		String p = getProperty(name);
 		if (p == null) throw new EntityPropertyNotDefinedException();
 		try
-		{
-			return TimeService.newTimeGmt(p);
+		{	TimeService timeService = ComponentManager.get(TimeService.class);
```
Comment (adrianfish) [inline kernel/kernel-util/src/main/java/org/sakaiproject/util/BaseResourceProperties.java]: Format's a bit odd here. Also, you may as well just put those on one line.

File: kernel/kernel-util/src/main/java/org/sakaiproject/util/BaseResourceProperties.java (hunk 1 above)
Comment (axxter99) [reply kernel/kernel-util/src/main/java/org/sakaiproject/util/BaseResourceProperties.java]: OK

File: kernel/kernel-util/src/main/java/org/sakaiproject/util/BaseResourceProperties.java (hunk 2 above)
Comment (axxter99) [reply kernel/kernel-util/src/main/java/org/sakaiproject/util/BaseResourceProperties.java]: OK

File: kernel/kernel-util/src/main/java/org/sakaiproject/util/BaseResourceProperties.java (hunk 1 above)
Comment (bjones86) [reply kernel/kernel-util/src/main/java/org/sakaiproject/util/BaseResourceProperties.java]: `Vector` is deprecated, can't we just make this an `ArrayList`?

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakaiproject/sakai/pullRequest?prid=4677019&bid=15648448)

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakaiproject/sakai/pullRequest?prid=4677019&bid=15648448)

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakaiproject/sakai/pullRequest?prid=4677019&bid=15648448)

**Comment by axxter99** [comment]
@adrianfish OK?

**Comment by axxter99** [comment]
@adrianfish @mpellicer cheers!

**Pull request merged**